src/main/kotlin/blackjack/model/Players.kt

```diff
@@ -0,0 +1,32 @@
+package blackjack.model
+
+class Players(players: List<Player>) {
+    private val players = players
+
+    fun size() = players.size
+
+    fun findPlayer(index: Int): Player {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    operator fun get(index: Int): Player {
```